graphql/resolvers.go

```diff
@@ -870,6 +870,25 @@ func (r *queryResolver) SiteBanner(ctx context.Context) (*restModel.APIBanner, e
 	return &banner, nil
 }
 
+func (r *queryResolver) Host(ctx context.Context, limit *int, page *int, hostID *string) (*restModel.APIHost, error) {
+	host, err := host.FindOneByIdOrTag(*hostID)
+	if err != nil {
+		return nil, InternalServerError.Send(ctx, fmt.Sprintf("Error Fetching host: %s", err.Error()))
+	}
+
+	if host == nil {
+		return nil, ResourceNotFound.Send(ctx, fmt.Sprintf("cannot find host with id %s", *hostID))
+	}
+
+	apiHost := &restModel.APIHost{}
+
+	err = apiHost.BuildFromService(hostID)
```

Comment:
it looks like we will need to call BuildFromService again with the running task to populate the APIHost.RunningTask field. Check it out here: https://github.com/evergreen-ci/evergreen/blob/master/rest/model/host.go#L71-L74

Reply:
The host struct already has a runningTask field. Does it not take the value from there? https://github.com/evergreen-ci/evergreen/blob/master/model/host/host.go#L57

Reply:
Nvm, I see what I need to do. fixed it.